docs/examples/academy/28. Aggregating Cargo Movements Data.ipynb

```diff
@@ -816,32 +768,18 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 13,
-   "metadata": {
-    "execution": {
-     "iopub.execute_input": "2021-07-13T11:16:25.092463Z",
-     "iopub.status.busy": "2021-07-13T11:16:25.091892Z",
-     "iopub.status.idle": "2021-07-13T11:16:25.095115Z",
-     "shell.execute_reply": "2021-07-13T11:16:25.095523Z"
-    }
-   },
+   "execution_count": 17,
+   "metadata": {},
    "outputs": [],
    "source": [
-    "quantity_by_destination = cms.groupby('unloading_country').sum()[['quantity']]\n",
+    "quantity_by_destination = cms.groupby('unloading_country').sum(numeric_only=True)[['quantity']]\n",
```

Comment:
This is the only meaningful change in this file - it fixes an issue where the `.sum` method was not working after a Pandas upgrade.